Blockzilla/TooltipView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  TooltipView.swift
```

Comment:
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

Use this header for all files